tactics/elim.ml

```diff
@@ -8,35 +8,148 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
+open CErrors
```

Comment:
Could we keep CErrors closed?

Reply:
OK, but what's the motivation?

Reply:
I can't put it in clear terms but I prefer it, so if the cost is low it would be nice

Reply:
Fine, I'll push an additional patch.